src/ln/channelmanager.rs

```diff
@@ -7501,7 +7501,7 @@ mod tests {
 			match $event[$event_idx] {
 				Event::SpendableOutputs { ref outputs } => {
 					match outputs[$output_idx] {
-						SpendableOutputDescriptor::DynamicOutput { ref outpoint, ref local_delayedkey, ref witness_script, ref to_self_delay } => {
+						SpendableOutputDescriptor::DynamicOutput { ref outpoint, ref key, ref witness_script, ref to_self_delay } => {
```

Comment:
Can you move this chunk to the commit where the DynamicOutput type changed so that you dont have intermediary commits that dont compile/fail tests?

Reply:
Ah sorry, addressed